views/list-jobs.html

```diff
@@ -0,0 +1,94 @@
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Home</title>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Century+Gothic&display=swap">
+    <title>Job Management System | Kainos</title>
+    <link rel="icon" href="../public/images/favicon.ico">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
+    <link rel="stylesheet" href="../public/css/styles.css">
+</head>
+
+    <nav class="navbar navbar-expand-lg bg-light sticky-top shadow-sm p-3 bg-white rounded">
+        <div class="container">
+            <a class="navbar-brand pb-2" href="./index.html">
+                <img src="../public/images/kainos-logo.png" alt="Kainos Logo" width="100" height="30">
```

Comment:
Doesn't look like this is correctly referenced based on the screenshot on the ticket description. Also there is no `kainos-logo.png` image in this PR.